gradle/libs.versions.toml

```diff
@@ -1,21 +1,22 @@
 [versions]
-kotlin = "1.5.31"
+kotlin = "1.6.0"
 save-core = "0.3.0-SNAPSHOT"
-ktor = "1.6.5"
-okio = "3.0.0-alpha.10"
+ktor = "1.6.7"
+okio = "3.0.0"
 serialization = "1.3.1"
 kotlinx-datetime = "0.3.1"
 kotlinx-coroutines = "1.5.2"
-kotlin-wrappers = "0.0.1-pre.265-kotlin-1.5.31"
-spring-boot = "2.6.0"
+# wrappers > pre.266 use also react-router v6, which has a lot of breaking changes
+kotlin-wrappers = "0.0.1-pre.266-kotlin-1.6.0"
+spring-boot = "2.6.1"
 spring-security = "5.6.0"
 spring-cloud = "3.1.0"
 junit = "5.8.2"
-ktoml = "0.2.7"
+ktoml = "0.2.9"
```

Comment:
Why do we use ktoml in cloud? Isn’t the logic in save?

Reply:
Hm, we actually don't use it. It must've appeared from copy-paste between projects. I'll remove it.